### PR TITLE
Created Dockerfile

### PR DESCRIPTION
**Pull request by BTaskaya**
docker
run

Docker file created #11

### CODE REVIEW

**Comment by amitkumarj441** [comment]
Awesome @BTaskaya :+1:

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN pip install --trusted-host pypi.python.org -r requirements-test.txt
+RUN pip install -e . && echo "x509 Installed"
+RUN echo "x509 Installed"
+WORKDIR tests/
+CMD ["python", "manage.py", "migrate"]
+CMD ["python", "manage.py", "createsuperuser"]
+CMD ["python", "manage.py", "runserver"]
+EXPOSE 8000
+
+ENV NAME django509
```
Comment (nemesisdesign) [inline Dockerfile]: the name should be `djangox509` as `x509` is the name of the standard

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM python:2-onbuild
+
+WORKDIR .
+RUN apt-get update && apt-get install -y \ 
+    sqlite3 \
+    libsqlite3-dev
+RUN pip install --trusted-host pypi.python.org -r requirements-test.txt
+RUN pip install -e . && echo "x509 Installed"
```
Comment (nemesisdesign) [inline Dockerfile]: why do we need `&& echo "x509 Installed"` if it's repeated in the subsequent line?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+RUN echo "x509 Installed"
```
Comment (nemesisdesign) [inline Dockerfile]: could you please change this to `RUN echo "django-x509 Installed"`?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,17 @@
+FROM python:2-onbuild
```
Comment (nemesisdesign) [inline Dockerfile]: is this python2 or python3? Ensure is python3. Python 2 is obsolete, we keep it only for backward compatibility with dinosaurs ;-)

**Comment by BTaskaya** [comment]
Squashed commits.

**Comment by nemesisdesign** [comment]
Thank you @BTaskaya :+1:

**Comment by amitkumarj441** [comment]
Good work @BTaskaya .

Looking forward to have more awesomeness from you :smile:

**Pull request merged**